plugins/web/opentelemetry-instrumentation-document-load/README.md

```diff
@@ -82,6 +82,30 @@ Because the browser does not send a trace context header for the initial page na
 </body>
 ```
 
+## Optional : Add custom attributes to document load span if needed
+
+If it is needed to add custom attributes to the document load span,and/or document fetch span and/or resource fetch spans, respective functions to do so needs to be provided
+as a config to the DocumentLoad Instrumentation as shown below. The attributes will be added to the respective spans
+before the individual are spans are ended. If the function throws an error , no attributes will be added to the span and
+the rest of the process continues.
+
+```js
+
+const addCustomAttributesToSpan = (span: Span) => {
+  span.setAttribute('<custom.attribute.key>','<custom-attribute-value>');
+}
+
+registerInstrumentations({
+  instrumentations: [
+    new DocumentLoadInstrumentation({
+      applyCustomAttributesOnDocumentLoadSpan: addCustomAttributesToSpan,
```

Comment:
These are REALLY verbose configuration names, which will NEVER be subject to any form of minification[1] and as document load is browser specific, I'd suggest a couple of things
- Collapse these into a "SINGLE" callback that gets passed a constant enum (number) value identifying the type of span it has been passed
- Make this name A LOT smaller, perhaps something like `spanInitializer` or somehing descriptive but smaller.

**[1]**: The only way to "compress" these would be to create const strings and use them as the lookups, but even then you still have the LOOOOONH names at least once.

Reply:
ie. just of the "names" of these config values you have added 121 bytes of vs something smaller like the (spanInitializer) (16 bytes), not to mention as a single config you could reuse a bunch of code further keeping the code payload smaller (but still readable)